iphone/TitaniumKit/TitaniumKit/Sources/API/TiBindingTiValue.m

```diff
@@ -292,7 +292,7 @@ JSValueRef TiBindingTiValueFromNSObject(JSContextRef jsContext, NSObject *obj)
 
     // Add "nativeStack" key
     NSArray<NSString *> *nativeStack = [(NSException *)obj callStackSymbols];
-    NSInteger startIndex = 3; // Starting at index = 4 to not include the script-error API's
+    NSInteger startIndex = 0;
```

Comment:
Why not just drop this variable then?

Reply:
Because in the if guard below we manually retrieve the native call stack, and in that case we know for sure that we can drop the first two entries.